{
	"schema_version": "1.6.0",
	"id": "MAL-2024-4943",
	"modified": "2024-10-24T01:01:58Z",
	"published": "2024-06-25T13:33:49Z",
	"aliases": [
		"SNYK-PYTHON-COLORARA-5885048"
	],
	"summary": "Malicious code in colorara (PyPI)",
	"details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
	"affected": [
		{
			"package": {
				"ecosystem": "PyPI",
				"name": "colorara",
				"purl": "pkg:pypi/colorara"
			},
			"versions": [
				"0.1"
			],
			"database_specific": {
				"source": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/pypi/colorara/MAL-2024-4943.json"
			}
		}
	],
	"references": [
		{
			"type": "WEB",
			"url": "https://github.com/DataDog/malicious-software-packages-dataset"
		},
		{
			"type": "ADVISORY",
			"url": "https://security.snyk.io/vuln/SNYK-PYTHON-COLORARA-5885048"
		}
	],
	"credits": [
		{
			"name": "ReversingLabs",
			"contact": [
				"https://www.reversinglabs.com"
			],
			"type": "FINDER"
		}
	],
	"database_specific": {
		"malicious-packages-origins": [
			{
				"id": "RLMA-2024-03723",
				"import_time": "2024-06-28T02:48:36.229433133Z",
				"modified_time": "2024-06-25T13:33:49Z",
				"sha256": "32d17549287246407c25d2ebeb402f98019f7496424607d76f81ab82837447ca",
				"source": "reversing-labs",
				"versions": [
					"0.1"
				]
			},
			{
				"id": "RLUA-2024-08014",
				"import_time": "2024-10-24T00:58:58.284016168Z",
				"modified_time": "2024-10-16T14:38:25Z",
				"sha256": "91924752d49b75fc2c76ff496b952c676ba39f86baefc8404a7eb314f981e622",
				"source": "reversing-labs"
			}
		]
	}
}
